tests/ctst/package.json

```diff
@@ -14,13 +14,13 @@
     "prometheus-query": "^3.2.0"
   },
   "devDependencies": {
-    "cli-testing": "github:scality/cli-testing.git#0.2.4",
+    "cli-testing": "github:scality/cli-testing.git#v0.2.6",
     "@typescript-eslint/eslint-plugin": "^5.45.0",
     "@typescript-eslint/parser": "^5.45.0",
     "babel-jest": "^29.3.1",
     "eslint": "^8.28.0"
   },
   "scripts": {
-    "lint": "eslint ."
+    "lint": "eslint ./world/Zenko.ts"
```

Comment:
To enforce it, we need to call it in the CI at some point, IMO in the workflow where ctst is built: https://github.com/scality/Zenko/blob/daa12076a4802461c60c760decc9e66cbe373597/.github/workflows/end2end.yaml#L320

Reply:
this was an oopsie on my part, didn't mean to commit this line. 

but yes, I will add the CI call in the last PR ZENKO-4525